sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusProcessorClient.java

```diff
@@ -116,20 +117,21 @@
      * a new set of sessions will be processed.
      */
     public synchronized void start() {
-        if (isRunning.getAndSet(true)) {
+        if (isRunning.getAndSet(true) && !isClosed.getAndSet(false)) {
```

Comment:
Do we throw an exception or let let the processor client to continue receive if start() is called after close()?

Reply:
We should allow calling start again after close is called.